Scala2-Functional-Programming/Week6/forcomp/src/main/scala/forcomp/Anagrams.scala

```diff
@@ -145,8 +190,8 @@ object Dictionary {
       sys.error("Could not load word list, dictionary file not found")
     }
     try {
-      val s = scala.io.Source.fromInputStream(wordstream)
-      s.getLines.toList
+      val s = Source.fromInputStream(wordstream)(Codec.UTF8)
+      s.getLines().toList
```

Comment:
As you probably understood, the `()` is not really needed

Reply:
I think this change was done by the IDE..